src/Uno.UI/UI/Xaml/Controls/Popup/PopupPanel.cs

```diff
@@ -188,5 +190,36 @@ private protected override void OnUnloaded()
 			base.OnUnloaded();
 			this.SetLogicalParent(null);
 		}
+
+		// TODO: pointer handling should really go on PopupRoot. For now it's easier to put here because PopupRoot doesn't track open popups, and also we
+		// need to support native popups on Android that don't use PopupRoot.
+		private void OnPointerPressed(object sender, PointerRoutedEventArgs args)
+		{
+			// Make sure we are the original source.  We do not want to handle PointerPressed on the Popup itself.
+			if (args.OriginalSource == this && Popup is { } popup
+			)
+			{
+				// The check is here because ContentDialogPopupPanel returns true for IsViewHit() even though light-dismiss is always
+				// disabled for ContentDialogs
+				if (popup.IsLightDismissEnabled)
+				{
+					ClosePopup(popup);
+				}
+				args.Handled = true;
+			}
+		}
```

Comment:
I would personally prefer to keep this in the `Popup` as this `PopupPanel` is somehow expected to be removed. Once removed we will have to handle the event the same way, but on the dismiss layer or the `Popup` content itself.

Reply:
If `PopupPanel` is removed, this code should move to `PopupRoot`, as noted in the comment added in the code. `PopupPanel` is not `PopupRoot` but it's 'close enough' for this purpose.